src/SignalR/clients/csharp/Http.Connections.Client/src/HttpConnection.cs

```diff
@@ -608,6 +607,7 @@ private HttpClient CreateHttpClient()
                     // Check if the key is User-Agent and remove if empty string then replace if it exists.
                     if (string.Equals(header.Key, Constants.UserAgent, StringComparison.OrdinalIgnoreCase))
```

Comment:
In WASM, the user would need to set "X-SignalR-User-Agent", whereas outside of WASM they need to set "User-Agent" to override the default we set. I could see us keeping this behavior, or saying "User-Agent" will apply to both.